packages/editor/src/browser/navigation/navigation-location-service.ts

```diff
@@ -165,12 +166,13 @@ export class NavigationLocationService {
     }
 
     /**
-     * Clears the navigation history.
+     * Clears the navigation and recently closed editors history.
```

Comment:
nit: a more generic documentation should be provided, since we do not want to list everything the method does in the documentation as you can simply read the implementation. Clearing the editor history, or clear history is sufficient?